helpers/tf_cfg.py

```diff
@@ -52,6 +53,54 @@ def bring_log_level(initial_lvl: Union[int, str]) -> int:
     return log_levels.get(int(initial_lvl), logging.DEBUG)
 
 
+class Flags(object):
+    """Class to store different dynamic flags (extra parameters)."""
+
+    def __init__(self):
+        self._with_sudo: bool = False
```

Comment:
Let's use `dataclass` here. You don't add extra actions to the setter/getter. As you said yourself

Reply:
Discussed over a phone, updated (removed for now).